lib/ui/views/login_view.dart

```diff
@@ -31,7 +31,13 @@ class _LoginViewState extends State<LoginView> {
       ViewModelBuilder<LoginViewModel>.reactive(
         viewModelBuilder: () => LoginViewModel(intl: AppIntl.of(context)),
         builder: (context, model, child) => Scaffold(
-            backgroundColor: AppTheme.etsLightRed,
+            backgroundColor: model.getAppTheme() == ThemeMode.light.toString()
+                ? AppTheme.etsLightRed
+                : model.getAppTheme() == ThemeMode.dark.toString()
+                    ? AppTheme.primaryDark
+                    : Theme.of(context).brightness == Brightness.light
+                        ? AppTheme.etsLightRed
+                        : AppTheme.primaryDark,
```

Comment:
What? XD
```
if appSetting.theme is light:
    etsLightRed
else:
    if appSetting.theme is dark:
        primaryDark
    else:
        if theme.brightness is light:
            etsLightRed
        else:
            primaryDark
```

It's a bit overkill no? XD `Theme.of(context).brightness` is always representing the actual theme of the application

```suggestion
            backgroundColor: Theme.of(context).brightness == Brightness.light
                        ? AppTheme.etsLightRed
                        : AppTheme.primaryDark,
```

This should be enough I think 😄